src/ngraph/runtime/gpu/cublas_emitter.cpp

```diff
@@ -211,6 +211,16 @@ size_t runtime::gpu::CUBLASEmitter::build_dot(const element::Type& dtype,
     return primitive_index;
 }
 
+size_t runtime::gpu::CUBLASEmitter::build_argmax(const element::Type& dtype,
+                                                 const Shape& arg0_shape,
+                                                 const Shape& out_shape,
+                                                 size_t axis,
+                                                 const Node* node)
+{
+    size_t primitive_index = 0;
```

Comment:
I think 0 could already been used. you can not just return 0.
 You can generate one with same process as others and use it with an empty implementation.